packages/block-editor/src/components/default-block-appender/index.native.js

```diff
@@ -20,6 +20,9 @@ import BlockInsertionPoint from '../block-list/insertion-point';
 import styles from './style.scss';
 import { store as blockEditorStore } from '../../store';
 
+const hitSlop = { top: 22, bottom: 22, left: 22, right: 22 };
```

Comment:
Is there a significance to these numbers or just what "felt" correct?

Reply:
These are the same hitSlop values used elsewhere throughout the app:
- [block-variation-picker](https://github.com/WordPress/gutenberg/blob/afc73eec673e59df108571869ce79eca7a7d6981/packages/block-editor/src/components/block-variation-picker/index.native.js#L33)
- [color-picker](https://github.com/WordPress/gutenberg/blob/afc73eec673e59df108571869ce79eca7a7d6981/packages/components/src/color-picker/index.native.js#L37)
- [embed-placeholder](https://github.com/WordPress/gutenberg/blob/afc73eec673e59df108571869ce79eca7a7d6981/packages/block-library/src/embed/embed-placeholder.native.js#L21)
- [media-placeholder](https://github.com/WordPress/gutenberg/blob/afc73eec673e59df108571869ce79eca7a7d6981/packages/block-editor/src/components/media-placeholder/index.native.js#L40)

I'm not sure where the `22` value specifically comes from, but it does appear to be a repeated standard that could likely be stored somewhere more centralized and exported for reuse in a separate PR. For my own understanding, do we have a separate file to store values like this (especially as hitSlop only relates to mobile)? Possibly somewhere in [components/src/utils](https://github.com/WordPress/gutenberg/tree/afc73eec673e59df108571869ce79eca7a7d6981/packages/components/src/utils), but I didn't see anything obvious that already exists.